region/norfair/west/Ice Beam Gate Room.json

```
@@ -1004,6 +1004,29 @@
         }
       }
     },
+    {
+      "link": [4, 4],
+      "name": "Leave Shinecharged",
+      "requires": [
+        "Morph",
+        {"canShineCharge": {
+          "usedTiles": 26,
+          "openEnd": 1
+        }},
+        "canShinechargeMovementTricky"
```

```suggestion
        "canShinechargeMovementComplex"
```
It's `canShinechargeMovementComplex` level with some extra obscurity points.  Very Hard players can figure it out.